### ISSUE TITLE
Setting up Traefik fails when machine does not have dynamically assigned IP

### ISSUE THREAD

**Issue by ernestasjuska** [microsoft/navcontainerhelper#542]
Hi,

Problem is in Setup-TraefikContainerForNavContainers. When machine has no dynamically assigned IP addresses it fails with this error:
```
Setup-TraefikContainerForBCContainers : The property 'IPAddress' cannot be found on this object. Verify that the property exists.
At C:\Users\ernjus\Desktop\trae.ps1:1 char:1
+ Setup-TraefikContainerForBCContainers -PublicDnsName bcapps.softera.l ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Setup-TraefikContainerForNavContainers], PropertyNotFoundException
    + FullyQualifiedErrorId : PropertyNotFoundStrict,Setup-TraefikContainerForNavContainers
```

```powershell
$IP = (Get-NetIPAddress -AddressFamily IPv4 | Where-Object PrefixOrigin -eq "Dhcp").IPAddress
```

~Variable $IP is not used anywhere as far as I can tell. Maybe it is enough to remove this statement?~

Thanks.

**Update**: No. It is actually used it next line to format toml. :)
Maybe add parameter to specify IP manually?

**Comment by freddydk** [comment]
@tfenster - should we add the IP as a parameter and default it like suggested?

**Comment by tfenster** [comment]
@freddydk makes sense, yes

**Comment by freddydk** [comment]
Fixed in 0.6.2.92
